ci-operator/step-registry/openshift/upgrade/azure/heterogeneous/OWNERS

```diff
@@ -0,0 +1,3 @@
+approvers:
```

Comment:
should we use multiarch-approvers and multiarch-reviewers for this?

Reply:
yeah good point. let me change it